packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -231,7 +233,12 @@ class NotificationDialog extends React.Component {
     }
 
     const defaultHeader = variant === variants.CUSTOM ? '' : <FormattedMessage id={`Terra.notification.dialog.${variant}`} />;
-    const notificationDialogClassNames = cx('notification-dialog', customProps.className);
+    const theme = this.context;
```

Comment:
We should be able to use the themeContext hook here

Reply:
Fixed

Reply:
did this change revert?